values.yaml

```diff
@@ -13,8 +13,10 @@ global:
 # Top lvl flat for easier maintenance
 images:
   graphdb:
+    registry: docker.io
     repository: ontotext/graphdb
-    tag: "10.5.1"
+    # If specified, overrides Chart.AppVersion
+    tag: ""
   busybox:
     repository: busybox
     tag: "1.31"
```

Comment:
the latest for this seems to be 1.36.1, I guess we can update it.

Reply:
Done